src/components/blocks/block.js

```diff
@@ -135,12 +135,12 @@ Block.prototype.getDefaultOutputLabel = function() {
 /**
  * Set the values
  */
-Block.prototype.setValues = function(values, json = false)
+Block.prototype.setValues = function(values, json = false, addNonActiveIfRef = false)
```

Comment:
For me it's not clear when to set the new parameter to true/false. Wondering whether this should always be true?